ovh/resource_ovh_ip_reverse.go

```diff
@@ -0,0 +1,154 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type OvhIpReverse struct {
+	IpReverse string `json:"ipReverse"`
+	Reverse   string `json:"reverse"`
+}
+
+func resourceOvhIpReverse() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceOvhIpReverseCreate,
+		Read:   resourceOvhIpReverseRead,
+		Update: resourceOvhIpReverseUpdate,
+		Delete: resourceOvhIpReverseDelete,
+
+		Schema: map[string]*schema.Schema{
+			"ip": {
+				Type:     schema.TypeString,
```

Comment:
ip shoud have a validate function which checks the the string is formatted as an ipv4 block